store/postgres/src/relational.rs

```diff
@@ -123,6 +123,9 @@ pub struct Layout {
     /// database tables that contain entities implementing
     /// that interface
     pub interfaces: HashMap<String, Vec<Rc<Table>>>,
+    /// Enums defined in the schema and their possible values. The names
+    /// are the original GraphQL names
+    pub enums: HashMap<String, Vec<String>>,
```

Comment:
Lets use a BTreeMap instead, otherwise the `create type` statements will be emitted in random order.

Reply:
Ok.